src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs

```diff
@@ -53,23 +54,55 @@ public PrintPreviewControl()
         ResetForeColor();
         Size = new Size(100, 100);
         SetStyle(ControlStyles.ResizeRedraw, false);
-        SetStyle(ControlStyles.Opaque | ControlStyles.OptimizedDoubleBuffer, true);
+        SetStyle(ControlStyles.AllPaintingInWmPaint | ControlStyles.UserPaint | ControlStyles.OptimizedDoubleBuffer, true);
         TabStop = false;
+
+        _hScrollBar.AccessibleName = SR.PrintPreviewControlAccHorizontalScrollBarAccName;
+        _hScrollBar.Anchor = AnchorStyles.Left | AnchorStyles.Bottom | AnchorStyles.Right;
+        _hScrollBar.Left = _focusHOffset;
+        _hScrollBar.SmallChange = ScrollSmallChange;
+        _hScrollBar.Visible = false;
+        _hScrollBar.ValueChanged += scrollBar_ValueChanged;
+        _hScrollBar.TabIndex = 0;
+        _hScrollBar.TabStop = true;
+        Controls.Add(_hScrollBar);
+
+        _vScrollBar.AccessibleName = SR.PrintPreviewControlAccVerticalScrollBarAccName;
+        _vScrollBar.Anchor = AnchorStyles.Top | AnchorStyles.Right | AnchorStyles.Bottom;
+        _vScrollBar.SmallChange = ScrollSmallChange;
+        _vScrollBar.Top = _focusVOffset;
+        _vScrollBar.Visible = false;
+        _vScrollBar.Scroll += scrollBar_ValueChanged;
```

Comment:
Should it be the ValueChanged event here?
```suggestion
        _vScrollBar.ValueChanged += scrollBar_ValueChanged;
```